exercises/secret-handshake/src/example.c

```diff
@@ -0,0 +1,58 @@
+#include "secret_handshake.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdbool.h>
+
+typedef struct {
+   const char *cmd;
+   const size_t num;
+} cmd_t;
+
+static const cmd_t cmds[] = {
+   {"wink", 1}, {"double blink", 2},
+   {"close your eyes", 4}, {"jump", 8}
+};
+
+static const size_t CMD_COUNT = ARRAY_SIZE(cmds);
+static const size_t CMD_REVERSE = 16;
+
+/* bitcount: count the number of set bits in n */
+static size_t bitcount(size_t num)
+{
+   size_t count;
+   for (count = 0; num; num = num & (num - 1), count++) ;
```

Comment:
Won't this just return the total number of bits (`count++` is executed on every iteration) rather than the number of bits that are 1?